conf/application.conf

```diff
@@ -126,6 +126,11 @@ microservice {
       host = localhost
       port = 1234
     }
+
+    pega {
+      host = localhost
+      port = 1234
+    }
   }
```

Comment:
I'm not sure what you mean by this? We can't set them like variables in application.conf

Reply:
Sorry, what is meant is the port value for 'pega' and 'etmp' being the same ie 1234.

Reply:
Ah! It's not important what it is locally because we override them in the environments, there is no meaning behind port 1234 (its just an arbitrary port)

Reply:
Sure, thanks. 😊